articles/relnotes/index.md

```diff
@@ -10,6 +10,14 @@ uid: microsoft.quantum.relnotes
 
 # Microsoft Quantum Development Kit Release Notes
 
+
+# Version 0.5.1904
+
+*Release date: Aprint 2015, 2019*
```

Comment:
April? 2019?

Reply:
Looks like something may have gone wrong with the last release notes, then? Pinging @cpalmer2020 for awareness.